network-api/networkapi/wagtailpages/templates/wagtailpages/article_page.html

```diff
@@ -26,7 +26,7 @@
           {# If there are h2s in the body of this page, show the article "table of contents" #}
           <div class="article-summary">
             <div class="article-summary-title body-small">{% trans "SUMMARY" %}</div>
-            <div class="article-links d-flex flex-wrap">
+            <div class="article-links">
```

Comment:
```suggestion
            <div class="article-links {% if get_titles|length > 5 %}article-links__two-col{% endif %}">
```
☝️ With that we wouldn't need any of the JavaScript and would be less for us to maintain in the future.

Reply:
Hi Kalob! Thanks for the awesome suggestion, I went and implemented this and removed the unnecessary JS, so I am marking these convos as resolved